motion_computation/config/parameters.yaml

```diff
@@ -31,3 +31,10 @@ enable_mobility_path_processing: false
 # Boolean: If true then ExternalObjects generated from sensor data will be processed.
 #          If other object sources are enabled, they will be synchronized but no fusion will occur (objects may be duplicated)
 enable_sensor_processing: false
+
+# Boolean: True if CTRV motion model should be used for the object type. False, if CV should be used:
+enable_ctrv_for_unknown_obj: true
```

Comment:
In `MotionComputationWorker::predictionLogic()`, we used to use the CV model for unknown objects. Is there a reason we are changing it to CTRV by default? Want to understand if there is a reason, and if it may change any previous Vulnerable Road User use case behavior.

Reply:
Originally, it was actually a little confusing because there were two different places unknown could be used. 1) If it was obj.UNKNOWN, it used CTRV. 2) but when enum didn't match anything such as UNKNOWN or PEDESTRIAN VEHICLE etc, it forced the type to UNKNOWN and used CV. So I just picked CTRV because that looked like what most instances would have executed.